sonarqube/pyproject.toml

```
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
     "hatchling>=0.11.2",
-    "setuptools<61",
+    "setuptools>=66",
```

`sonarqube` is still present in the Agent's Python 2 embedded environment, so this should be:
```suggestion
    "setuptools>=66; python_version > '3.0'",
    "setuptools; python_version < '3.0'",
```